librz/core/cmd/cmd_debug.c

```diff
@@ -3415,3 +3305,55 @@ RZ_IPI RzCmdStatus rz_cmd_debug_window_identify_handler(RzCore *core, int argc,
 	return RZ_CMD_STATUS_ERROR;
 #endif
 }
+
+RZ_IPI RzCmdStatus rz_cmd_debug_esil_add_handler(RzCore *core, int argc, const char **argv) {
+	int perm = rz_str_rwx(argv[1]);
+	int kind = (int)argv[2][0];
```

Comment:
this is not understandable, but... it is what it was :(

Reply:
I opted not to clean this code too much since it will be deleted sometime in the future anyway. Along with `librz/debug/desil.c`.